src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyTotalAmount = 0;
+        int buyTotalAmount = 0;
+        for (String line: readFromFile(fromFileName).split("\\r?\\n")) {
```

Comment:
here you can split by System.lineSeparator if you follow my previous advice while reading